spec/factories/users.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :administrator, class: User do
+    name { 'factory_name1' }
+    email { 'admin@factory.com' }
+    password { 'aaaaaa' }
```

Comment:
存在しないドメインである`@example.com`を使用するべきです。
また、細かいですがパスワードは特に指定がない限りはすべて同じ文字列で固定した方が扱いやすいと思います。
```suggestion
    email { 'admin@example.com' }
    password { 'password' }
```

Reply:
修正済みです。 817284c